engine/execution/state/delta/delta.go

```diff
@@ -36,6 +36,15 @@ func (d Delta) Set(owner, key string, value flow.RegisterValue) {
 	d.Data[k] = value
 }
 
+// UpdatedRegisterIDs returns all register ids that were updated by this delta.
+func (d Delta) UpdatedRegisterIDs() []flow.RegisterID {
```

Comment:
For certain use cases it is very important that we know that these are not deterministically sorted.
Could you add to the comment that the order of these is not deterministic.

Reply:
good point.  i'm thinking of making fvm/state/View interface the primary view interface.  i'll add the comment there

Reply:
shouldn't the sorting be done within the ledger / NewUpdate in that case?

Reply:
We could. We need to make the returned slice to be deterministic, so that there is no other side effect. And I just wanted to point out your change can't ensure the order is deterministic.